server/bin/gold/test-30.1.txt

```diff
@@ -0,0 +1,185 @@
+-rw-rw-r--      11798 public_html/static/css/v0.3/css/LICENSE.TXT
+-rw-rw-r--       3663 public_html/static/css/v0.3/css/jschart.css
+drwxrwxr-x          - public_html/static/js
+drwxrwxr-x          - public_html/static/js/v0.2
+drwxrwxr-x          - public_html/static/js/v0.2/js
+-rw-rw-r--       9415 public_html/static/js/v0.2/js/app.js
+-rw-rw-r--       5556 public_html/static/js/v0.2/js/pbench_utils.js
+drwxrwxr-x          - public_html/static/js/v0.3
+drwxrwxr-x          - public_html/static/js/v0.3/js
+-rw-rw-r--      11798 public_html/static/js/v0.3/js/LICENSE.TXT
+-rw-rw-r--     143934 public_html/static/js/v0.3/js/jschart.js
+drwxrwxr-x          - public_html/users
+--- pbench-satellite tree state
++++ pbench-satellite-local tree state (/var/tmp/pbench-test-server/test-30.1/pbench-satellite-local)
+drwxrwxr-x          - logs
+drwxrwxr-x          - pbench-move-results-receive
+drwxrwxr-x          - pbench-move-results-receive/fs-version-002
+drwxrwxr-x          - quarantine
+drwxrwxr-x          - quarantine/duplicates-002
+drwxrwxr-x          - quarantine/errors-002
+drwxrwxr-x          - quarantine/md5-002
+drwxrwxr-x          - tmp
+--- pbench-satellite-local tree state
++++ pbench log file contents
+++++ pbench-local/logs
++++++ pbench-audit-server/pbench-audit-server.error
+----- pbench-audit-server/pbench-audit-server.error
++++++ pbench-audit-server/pbench-audit-server.log
+1970-01-01T00:00:42.000000 INFO pbench-audit-server.report post_status -- @cee:{"@generated-by": {"commit_id": "unit-test", "group_id": 43, "hostname": "example.com", "pid": 42, "user_id": 44, "version": ""}, "@timestamp": "1970-01-01T00:00:42", "chunk_id": 1, "doctype": "status", "name": "pbench-audit-server", "text": "pbench-audit-server.run-1970-01-01T00:00:42-UTC(unit-test)\n", "total_chunks": 1, "total_size": 59}
+----- pbench-audit-server/pbench-audit-server.log
++++++ pbench-server-put-shim-001/pbench-server-put-shim-001.log
+1970-01-01T00:00:42.000000 INFO pbench-server-put-shim-001.pbench-server-put-shim-001 process_tb -- run-1970-01-01T00:00:42-UTC
+1970-01-01T00:00:42.000000 INFO pbench-server-put-shim-001.pbench-server-put-shim-001 process_tb -- tarball-0_1970.01.01T00.42.00.tar.xz: OK
+1970-01-01T00:00:42.000000 INFO pbench-server-put-shim-001.pbench-server-put-shim-001 process_tb -- tarball-w-dot-prefix_1970.01.01T00.42.00.tar.xz: OK
+1970-01-01T00:00:42.000000 INFO pbench-server-put-shim-001.pbench-server-put-shim-001 process_tb -- tarball-w-prefix-dot_1970.01.01T00.42.00.tar.xz: OK
+1970-01-01T00:00:42.000000 INFO pbench-server-put-shim-001.pbench-server-put-shim-001 process_tb -- tarball-duplicate_1970.01.01T00.42.00.tar.xz: OK
+1970-01-01T00:00:42.000000 INFO pbench-server-put-shim-001.pbench-server-put-shim-001 process_tb -- tarball-bad-md5_1970.01.01T00.42.00.tar.xz: OK
+1970-01-01T00:00:42.000000 INFO pbench-server-put-shim-001.pbench-server-put-shim-001 process_tb -- tarball-bad-md5-1_1970.01.01T00.42.00.tar.xz: OK
```

Comment:
The unittest "mock" action just returns 0, but this looks weird and doesn't resemble what we'd see in an actual run.

I'm debating how much to be concerned about this, but it _would_ be a good thing to have the test output confirm that we handle an upload error cleanly and a slightly more "real" mock would allow that. Since the tarballs are well labeled, it could even be a match for `bad-md5` in the filename, although it wouldn't be too hard to do better than that.

Reply:
I moved in pytest direction and removed this unit test. Sorry about that